### DISCUSSION TITLE
dicom images

### DISCUSSION THREAD

**Discussion by shahriar-faghani** [Project-MONAI/MONAI#1586] [Q&A]
I have problem with loading dicom images with monai, can somebody help me?

**Comment by shahriar-faghani** [comment]
' can not find suitable reader for this file: F:\monai\test_image.dcm.'

**Comment by Nic-Ma** [comment]
Hi @shahriar-faghani ,

May I know what your MONAI version is? We started to support DICOM from v0.4.
You can refer to this tutorial for more details: https://github.com/Project-MONAI/tutorials/blob/master/modules/load_medical_images.ipynb

Thanks.

**Reply by shahriar-faghani** [reply]
Hi @Nic-Ma 
Thanks for your response, My monai version is 0.4!

**Reply by rijobro** [reply]
Can you run the following command and paste the output here?
```bash
python -c 'from monai.config import print_config; print_config()'   
```

**Reply by shahriar-faghani** [reply]
ONAI version: 0.4.0
Numpy version: 1.19.2
Pytorch version: 1.6.0
MONAI flags: HAS_EXT = False, USE_COMPILED = False
MONAI rev id: 0563a4467fa602feca92d91c7f47261868d171a1

Optional dependencies:
Pytorch Ignite version: NOT INSTALLED or UNKNOWN VERSION.
Nibabel version: 3.2.1
scikit-image version: 0.18.1
Pillow version: 7.2.0
Tensorboard version: 2.3.0
gdown version: NOT INSTALLED or UNKNOWN VERSION.
TorchVision version: NOT INSTALLED or UNKNOWN VERSION.
ITK version: 5.1.2
tqdm version: 4.54.1
lmdb version: NOT INSTALLED or UNKNOWN VERSION.
psutil version: NOT INSTALLED or UNKNOWN VERSION.

For details about installing the optional dependencies, please visit:
    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies

**Reply by rijobro** [reply]
This looks ok to me. Can you double check that a file exists in the path: `F:\monai\test_image.dcm`?
